packages/terra-notification-dialog/src/terra-dev-site/doc/example/example-styles.module.scss

```diff
@@ -1,10 +1,23 @@
+@import '~terra-mixins/lib/Mixins';
+
 :local {
   .example-wrapper {
-    border: 1px solid #d3d3d3;
-    height: 28rem;
+    height: 3rem;
   }
 
   .content-wrapper {
     padding: 0.7rem;
   }
+
+  .custom-notification-dialog-icon {
+    background: var(--terra-notification-dialog-example-custom-icon-background, inline-svg("<svg viewBox='0 0 49 49' xmlns='http://www.w3.org/2000/svg'><path fill='#654A08' d='M24 2.3L45.7 24 24 45.7 2.3 24 24 2.3M24 0c-.3 0-.5.1-.8.3L.3 23.2c-.4.5-.4 1.1 0 1.6l22.9 22.9c.2.2.5.3.8.3s.5-.1.8-.3l22.9-22.9c.4-.5.4-1.1 0-1.6L24.8.3c-.3-.2-.5-.3-.8-.3zm0 8c3.9 3.9 4 13 4 13H15v6c0 10 9 13 9 13-3.9-3.9-4-13-4-13h13v-6c0-10-9-13-9-13z' ></path></svg>"));
```

Comment:
Switch these icon examples to the purple IconDiamond instead of IconAbnormal

Reply:
updated: https://github.com/cerner/terra-framework/pull/1105/commits/c78f842fd0620606d38c4b19cf5cdfc8c76e50e0#diff-5e62aa72fdf94aca7bf79ccb953098feR13